ex2_system/source/leop/leop.c

```diff
@@ -37,64 +38,74 @@
 bool deploy_all_deployables() {
     Deployable_t sw;
 
-    int getStatus_retries;
+    int deployment_attempt;
     uint16_t burnwire_currents[8] = {0};
 
-    for (getStatus_retries = 0; getStatus_retries <= MAX_RETRIES; getStatus_retries++) {
+    for (deployment_attempt = 1; deployment_attempt <= MAX_ATTEMPTS; deployment_attempt++) {
         sw = DFGM;
         // Deploy DFGM
-        if ((switchstatus(sw) != 1) && (getStatus_retries != MAX_RETRIES)) {
-            ex2_log("Check #%d: %c not deployed\n", &getStatus_retries, sw);
-            ex2_log("Activated %c\n", sw);
-            activate(sw);
-            vTaskDelay(TWENTY_SEC_DELAY);
-        } else if ((switchstatus(sw) != 1) && (getStatus_retries == MAX_RETRIES)) {
-            ex2_log("Check #%d: %c not deployed, exiting the LEOP sequence.\n", &getStatus_retries, sw);
-            return false;
-        }
+        ex2_log("Activated burnwire %c\n", sw);
```

Comment:
Why did you remove the part where the code checks the deployment status?

Reply:
mhm, we should include it here. ex2_log has been changed to sys_log since then as well

Reply:
What if the switch gives a faulty reading, or the deployable moves by 1mm to de-trigger the switch? Then boom, no antenna deployment and our satellite is dead to us.

Reply:
original design was to start over again if we get stuck deploying one switch (since they needed to be done in sequence so different deployables don't get in each other's way). I'm okay with simplifying it to 5 attempts and moving on if it's not successful.